doc/source/build_hints.rst

```diff
@@ -1819,6 +1878,10 @@ The following options are available to select a subset of drivers:
     activation of individual drivers. It might be needed to pass
     ``-UGDAL_ENABLE_DRIVER_* -UOGR_ENABLE_DRIVER_*`` to reset their state.
 
+    .. note::
+
+        The following GDAL drivers cannot be disabled: VRT, DERIVED, GTiff, COG, HFA, MEM.
+        The following OGR drivers cannot be disabled: "ESRI Shapefile", "MapInfo File", OGR_VRT, Memory, KML, GeoJSON, GeoJSONSeq, ESRIJSON, TopoJSON.
```

Comment:
Should "MITAB" in https://gdal.org/drivers/vector/index.html be changed to "MapInfo File"?

Reply:
yes